src/Vex/CommandContext.cpp

```diff
@@ -4,13 +4,14 @@
 #include <cmath>
 #include <variant>
 
+#include <Vex/BuiltInShaders/DepthStencilReadback.h>
```

Comment:
Looks like you ended up removing this feature from the readback (which is fair). In that case, this built-in shader should be removed.